WalletWasabi.Fluent/Views/AddWallet/LegalDocumentsView.axaml

```diff
@@ -11,7 +11,7 @@
              x:Class="WalletWasabi.Fluent.Views.AddWallet.LegalDocumentsView">
   <c:ContentArea Title="{Binding Title}"
                  Caption="Please read the small print."
-                 EnableNext="True" NextContent="Continue" FocusNext="True">
-      <TextBlock TextWrapping="Wrap" Text="{Binding Content}" />
+                 EnableBack="True">
```

Comment:
Can you add `FocusCancel="True" here?

If the user presses the Enter key it opens a new legalDocumentsView every time, because the focus stays on the opening button.

Reply:
ACK